### PR TITLE
update yolov5 notebook on nncf

### PR DESCRIPTION
**Pull request by eaidova**
111 notebook removed as it demonstates syntetic use case (quantize intel model from omz which already have in8 model and validate it on synthetic dataset different from used in original model validation), so I move yolov5 as good example how to quantize real model on its place. The notebook also contains comparison between POT old approach to NNCF.

Currently quantize_with_accuracy_control is out of scope of this notebook (but we can add it based on different model, where it is required and does not need large efforts)

### CODE REVIEW

**Review by alexsu52 — COMMENTED** [review]
LGTM, the https://github.com/openvinotoolkit/nncf/pull/1733 was merged to NNCF. It made PTQ NGraph implementation by default for OpenVINO backend. Please check once more that the notebook shows the same metrics.

File: notebooks/111-yolov5-quantization-migration/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+![Ultralytics Yolov5 results](https://user-images.githubusercontent.com/44352144/177097174-cfe78939-e946-445e-9fce-d8897417ef8e.png)
+
+
+This tutorial demonstrates how to migrate quantization pipeline written using the OpenVINO [Post-Training Optimization Tool (POT)](https://docs.openvino.ai/latest/pot_introduction.html) to [NNCF Post-Training Quantization API](https://docs.openvino.ai/latest/nncf_ptq_introduction.html). This tutorial is based on  [Ultralytics Yolov5](https://github.com/ultralytics/yolov5) model and additionally it compares model accuracy between the FP32 precision and quantized INT8 precision models and runs a demo of model inference based on sample code from [Ultralytics Yolov5](https://github.com/ultralytics/yolov5) with the OpenVINO backend.
+
+
+## Notebook Contents
+
+The tutorial consist from following parts:
```
Comment (MaximProshin) [inline notebooks/111-yolov5-quantization-migration/README.md:11]: The tutorial consist from following parts: => The tutorial consist**s** from **the** following parts:

File: notebooks/111-yolov5-quantization-migration/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Migrate quantization from POT API to NNCF
```
Comment (MaximProshin) [inline notebooks/111-yolov5-quantization-migration/README.md:1]: I would make it consistent: 
Migrate quantization from POT API to NNCF => Migrate quantization from POT API to NNCF API

**Review by MaximProshin — COMMENTED** [review]
LGTM. A few editorial suggestions

**Comment by MaximProshin** [comment]
As the notebook demonstrates how to use NNCF instead of POT, I think it's fair if INT8 accuracy and performance will be executed for both POT and NNCF versions. Have you try to collect it? Are they different?

File: notebooks/111-yolov5-quantization-migration/README.md (hunk 2 above)
Comment (eaidova) [reply notebooks/111-yolov5-quantization-migration/README.md]: done

**Comment by raymondlo84** [comment]
https://github.com/openvinotoolkit/openvino_notebooks/blob/main/README_cn.md 

@zhuo-yoyowz @eaidova  -- the new commit erased our changes to the AI trends part. Please cross-check.

**Pull request merged**